### ISSUE TITLE
SaplingGenerator is misleading

### ISSUE THREAD

**Issue by enbrain** [asiekierka/MinecraftPOMF#2866]
It does not generate a sapling. It generates a tree when a sapling grows naturally or when a bone meal item is used on a sapling.

**Comment by apple502j** [comment]
True, but it also does not generate a tree (you'd modify a configure feature for that.) `SaplingBehavior` maybe? does it sound like pistons too much?

**Comment by enbrain** [comment]
> `SaplingBehavior` maybe? does it sound like pistons too much?

I think sapling behavior is `SaplingBlock` itself. `PistonBehavior` might be an inaccurate name as well.

Maybe `SaplingGrowthBehavior` or `TreeGrowthBehavior`?

**Comment by apple502j** [comment]
Okay, I completely forgot PistonBehavior was just an enum :P

**Comment by haykam821** [comment]
There is also `CauldronBehavior`, which is an interface.
